test/base/module/record/Record.deleteAllRecords.test.js

```diff
@@ -0,0 +1,44 @@
+import Auth from '../../../../src/node/authentication/Auth';
+import Connection from '../../../../src/node/connection/Connection';
+import Record from '../../../../src/base/module/record/Record';
+import {URI, USERNAME, PASSWORD, DOMAIN} from './common';
+import nock from 'nock';
+import KintoneAPIException from '../../../../src/base/exception/KintoneAPIException';
+
+const auth = new Auth();
+auth.setPasswordAuth({username: USERNAME, password: PASSWORD});
+const connection = new Connection({auth, domain: DOMAIN});
+const recordModule = new Record({connection});
+
+const BULK_REQUEST_API_ROUTE = '/k/v1/bulkRequest.json';
+
+describe('Check Record.deleteAllRecords', () => {
+  it('should throw error when called with invalid app ID', () => {
+    return recordModule.deleteAllRecords(-1, []).catch((err)=>{
+      // TODO: deleteAllRecords line 380 is unnecessary
+      expect(err).toBeInstanceOf(KintoneAPIException);
```

Comment:
@phongnhat19 
please, nock and verify error correctly.
currently error is KintoneAPIException [Error]: getaddrinfo ENOTFOUND my_domain my_domain:443